tests/test_rules.py

```diff
@@ -65,6 +65,7 @@ def get_rule_yaml(self, file_path: str) -> dict:
         # "There are rule files with extensions other than .yml")
 
     def test_legal_trademark_violations(self):
+        # See Issue # https://github.com/SigmaHQ/sigma/issues/1014
```

Comment:
1014 is Develop Sigma rules for Living Off The Land Binaries and Scripts ???

Reply:
fixed the link